### PR TITLE
openstack UPI: Remove Swift example, use Glance

### PR DESCRIPTION
**Pull request by pierreprinetti**
The UPI workflow needs the bootstrap ignition file to be hosted
somewhere. Prior to this patch, the first proposed example used Swift.
However, the example was using commands that are not available in the
openstack client shipped with the recommended operating systems and
repositories.

This patch removes the Swift example and focuses the workflow on using
Glance instead, along the lines of what the IPI installer does. The UPI
docs still mention that any hosting system can be used, including Swift.

### CODE REVIEW

File: docs/user/openstack/install_upi.md [hunk 1]
```diff
@@ -617,34 +617,16 @@ In this guide, we will assume the file is at the following URL:
 
 https://static.example.com/bootstrap.ign
 
-##### Example 1: Swift
+##### Example: Glance image service
```
Comment (mandre) [inline docs/user/openstack/install_upi.md:620]: Please also update the Table of Contents. I think we could remove the example links from it.

File: docs/user/openstack/install_upi.md [hunk 2]
```diff
@@ -692,6 +671,12 @@ Create a file called `$INFRA_ID-bootstrap-ignition.json` (fill in your `infraID`
     "config": {
       "merge": [
         {
+	  "httpHeaders: [
```
Comment (mandre) [inline docs/user/openstack/install_upi.md:674]:
```suggestion
	  "httpHeaders": [
```

File: docs/user/openstack/install_upi.md [hunk 3]
```diff
@@ -941,7 +926,14 @@ The teardown playbook deletes the bootstrap port and server.
...
 
-If you haven't done so already, you should also disable the bootstrap Ignition URL.
+If you haven't done so already, you should also disable the bootstrap Ignition URL. If you are following the Glance example:
+
+<!--- e2e-openstack-upi: INCLUDE START --->
+```sh
+$ openstack image delete "bootstrap-ign-${INFRA_ID}"
+```
+<!--- e2e-openstack-upi: INCLUDE END --->
+
```
Comment (mandre) [inline docs/user/openstack/install_upi.md:936]: We should probably also document revoking the glance token.

**Comment by pierreprinetti** [comment]
Updated the TOC, fixed JSON syntax and added token revocation. Thanks @mandre !

**Review by mandre — COMMENTED** [review]
Looking good now, thanks!
/lgtm

**Pull request merged**